pthread-makefile-examples/README.md

```diff
@@ -0,0 +1,66 @@
+# Pthread and Makefile Tutorial
+
+This directory contains simple programs that serves as a tutorial for pthreads and makefile. \
+
+## 1. Pthread: Creating threads. Makefile: Target, Dependencies, Recipes and Rules
+
+### Brief 
+This directory contains two programs, 1_thread_create.c and 2_thread_create.c. 
+
+1_thread_create.c is used to create a thread, "threadFunc" which prints the process ID of the thread. \
+It is important to note that the process ID of main thread and threadFunc are the same as threads run within the context of a single process. \
+
+2_thread_create.c is used to introduce how to terminate a thread using pthread_exit(). 
+
+### Build
+Executable files can be generated by the use of the provided makefile. \
+This makefile explains the following concepts with examples: Targets, Dependencies, Recipes and Rules. 
+
+### Expected Output
+
+1_thread_create.c: 
+
+![image](https://user-images.githubusercontent.com/43494309/164957026-de26db42-59d6-419c-ba3b-b5e0fd2f9021.png)
```

Comment:
@DeekshithPatil favor inline comments instead of images.  The images don't really work unless you look on github, so you can't cat out the README and copy/paste into the command line.